server/handlers.go

```diff
@@ -209,6 +209,53 @@ func (s *Server) discoveryHandler() (http.HandlerFunc, error) {
 	}), nil
 }
 
+func (s *Server) isFailedAttemptDurationExceeded(u storage.BlockedUser) bool {
```

Comment:
definitely, this code is not easy to maintain, at least the terminology is not proper.
FYI - we are using the BlockedUser instance at most of the places, but that is not a blocked user, that is just an invalid attempt, user will be blocked when he exceeds the max invalid attempts. 

Either please change the terminology or provide more comments.

Reply:
Sure, will change the terminology